test/specs/mainnet/transactionHistory/arbitrum.spec.js

```diff
@@ -123,7 +124,7 @@ describe('The PrimeSDK, when get the single transaction and multiple transaction
     }, data.retry); // Retry this async test up to 5 times
   });
 
-  it('SMOKE: Validate the transaction history of the native token transaction on arbitrum network', async function () {
+  it('SMOKE: Validate the transaction history of the native token transaction on the arbitrum network', async function () {
```

Comment:
Can we also include a test to validate the response type from getTransactionHistory doesn't change?

Reply:
Yeah, We can add response validation of the getTransactionHistory function.